src/main/java/org/apache/sling/distribution/journal/kafka/KafkaClientProvider.java

```diff
@@ -79,6 +79,10 @@
 
     public static final int PARTITION = 0;
 
+    private static final String SECURITY_PROTOCOL = "security.protocol";
```

Comment:
Couldn't we import the constants directly from `org.apache.kafka.clients.consumer.ConsumerConfig` and `org.apache.kafka.clients.producer.ProducerConfig` rather than redefining them ?

Reply:
I think I did not find them.. Will try.

Reply:
Good point. Those seem to be defined in `org.apache.kafka.common.config.SaslConfigs` (`SASL_MECHANISM` and `SASL_JAAS_CONFIG`)

Reply:
I forgot `CommonClientConfigs` contained the field `SECURITY_PROTOCOL_CONFIG` that should replace the `SECURITY_PROTOCOL` constant.

Reply:
Done